bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -98,6 +98,7 @@ import Chainweb.Version.Utils
 
 import Data.CAS.HashMap hiding (toList)
 import Data.CAS.RocksDB
+import qualified Database.RocksDB.Internal as R
```

Comment:
Is this required? I would expect the `R` qualifier to be used somewhere in the file.